pkg/kudoctl/kudoinit/manager/manager.go

```diff
@@ -33,8 +34,8 @@ type Initializer struct {
 }
 
 // NewInitializer returns the setup management object
-func NewInitializer(options kudoinit.Options) Initializer {
-	return Initializer{
+func NewInitializer(options kudoinit.Options) *Initializer {
```

Comment:
is there a reason for this change?
makes me wonder if the methods need adjustment as well... meaning 
`func (m Initializer) UninstallService(client *kube.Client) error {` ->
`func (m *Initializer) UninstallService(client *kube.Client) error {` ?

Reply:
Mostly to make the usage of this more clear in `setup.go`. And i've changed the receiver to a pointer. It's not required to be mutable right now, but it might as well be in the future and can be hard to find these kind of bugs :-/